### ISSUE TITLE
Cookie Button doesn't have a sufficient contrast ratio

### ISSUE THREAD

**Issue by sir-red-dab** [sir-red-dab/jobrodo#20]
The button of the CookieBar doesn't have a sufficient contrast ratio. Make the background of the CookieBar the following color `#1F1F1F` and add the following background color to the button `#FFF`.
